### PR TITLE
shiftstack/shiftstack-ci: Add a cleanup periodic

### PR DESCRIPTION
**Pull request by pierreprinetti**
/hold

### CODE REVIEW

**Comment by pierreprinetti** [comment]
/hold cancel
Is this why it's not testing?

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+- dockerfile_literal: |-
+    FROM src
+    ADD openshift-install /usr/bin/openshift-install
+  from: src
+  inputs:
+    installer:
+      paths:
+      - destination_dir: .
+        source_path: /go/bin/openshift-install
+  to: installer
```
Comment (petr-muller) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:21]: Ideally call the resulting image something else than `installer`, installer is the name of the image that you import already.

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml [hunk 2]
```diff
@@ -1,8 +1,24 @@
+base_images:
+  installer:
+    name: installer
+    namespace: ocp
+    tag: latest
```
Comment (petr-muller) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:5]: There's no `ocp/installer:latest`. OCP images are collected via version-named imagestreams and tags are individual images in them. Example:

```
name: "4.8"
namespace: ocp
tag: installer
```

**Review by mdbooth — APPROVED** [review]

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml [hunk 3]
```diff
@@ -15,6 +31,12 @@ tests:
   commands: shellcheck ./clean-ci-resources.sh
   container:
     from: src
+- as: cleanup-vexxhost
+  cron: '@hourly'
```
Comment (mdbooth) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:35]: I'm not entirely sure how pj-rehearse deals with a cron job. The test seems to fail with no output, though. Do you have a clean execution of it already?

Comment (pierreprinetti) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml:35]: nope, I forgot to set the `/hold` flag. Not a big deal though, and thanks for the LGTM

**Pull request merged**